nextcord/interactions.py

```diff
@@ -407,6 +407,7 @@ async def send(
         tts: bool = False,
         ephemeral: bool = False,
         delete_after: Optional[float] = None,
+        allowed_mentions: Optional[AllowedMentions] = MISSING,
```

Comment:
This & other instances of this should default to None? None does not have a special case here (like resetting nicks) so None should be used?

Reply:
In the send functions for sync.py and async.py its `allowed_mentions: AllowedMentions = MISSING`. Should I change this to `allowed_mentions: Optional[AllowedMentions] = None` aswell? or just instances where AllowedMentions is already in an optional type.

Reply:
Weirddddd..... lets just leave it as is for now and solve it in v3